src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Interop/msquic_extensions.cs

```diff
@@ -0,0 +1,29 @@
+#pragma warning disable IDE0073
+//
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+//
+#pragma warning restore IDE0073
+
+using System;
+
+namespace Microsoft.Quic
+{
+    internal static unsafe class MsQuicExtensions
+    {
+        public static void SetConnectionCallback(this ref QUIC_API_TABLE Table, QUIC_HANDLE* Handle, delegate* unmanaged[Cdecl]<QUIC_HANDLE*, void*, QUIC_CONNECTION_EVENT*, int> Callback, void* Context)
```

Comment:
Should we also update our code to use these, or can it wait to be opportunistically included in some other PR?